app/src/components/common/BalanceRow.vue

```diff
@@ -66,6 +75,7 @@
     </div>
 
     <SendModal ref="SendModal" :denoms="[balance.denom]" />
+    <DelegationModal ref="StakeModal" />
```

Comment:
all these modals get mounted per balance row. feels unnecessary. it should be in TmBalance and a balance row should trigger an event that TmBalance is listening to. let's please refactor this right after this PR

Reply:
Or we can change it here? Ah no, OK, better individual PRs

Reply:
better another PR. but let's do it while we are aware.

Reply:
oki